src/gui/mzroll/mzfileio.cpp

```diff
@@ -1175,9 +1175,18 @@ PeakGroup* mzFileIO::readGroupXML(QXmlStreamReader& xml, PeakGroup* parent)
         if (matches.size() > 0)
             group->compound = matches[0];
     } else if (!compoundId.empty()) {
-        Compound* c = DB.findSpeciesByIdAndName(compoundId,
-                                                group->compound->name,
-                                                DB.ANYDATABASE);
+        Compound* c;
```

Comment:
It is a good practice to initialize variables (`nullptr` in this case) to a default value, since we cannot rely on the compiler to assign it our expected default.

Reply:
Updated.